common.blocks/i-bem/i-bem.bemtree

```diff
@@ -92,16 +92,23 @@ BEMContext.prototype.doAsync = function doAsync(fn) {
     return promise;
 };
 
-// Wrap xjst's apply and export our own
-var oldApply = exports.apply;
+/**
+ * waits for all, even nested, promises inside bemtree
+ * @param ctx
+ * @returns {Promise}
+ */
+var waitForPromises = function (ctx) {
```

Comment:
it's better to do `function waitForPromises(ctx) {` and avoid to change 95-96 lines